homeassistant/components/climate/radiotherm.py

```diff
@@ -29,13 +29,38 @@
 DEFAULT_AWAY_TEMPERATURE_HEAT = 60
 DEFAULT_AWAY_TEMPERATURE_COOL = 85
 
+# Mappings from radiotherm json data to HASS state flags.  Note that
```

Comment:
I couldn't find the reverse mappings you refer to. However, if they are somewhere, wouldn't it be better to generate them dynamically (e.g. via `{v: k for k, v in NAME_TEMP_MODE.items() }`)?